### PR TITLE
build(deps-dev): use new @fastify org dependencies

### PR DESCRIPTION
**Pull request by Fdawgs**
#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Comment by mcollina** [comment]
CI is failing

File: package.json [hunk 1]
```diff
@@ -30,9 +30,9 @@
   ],
   "license": "MIT",
   "devDependencies": {
+    "@fastify/jwt": "^5.0.0",
```
Comment (Eomm) [inline package.json:33]: This includes the v4:
https://github.com/fastify/fastify-jwt/releases/tag/v4.0.0

We should keep the v3 till fastify v4

**Comment by mcollina** [comment]
CI is failing

File: package.json (hunk 1 above)
Comment (Fdawgs) [reply package.json:33]: Cheers @Eomm, will close this for now and revisit with v4's release

Comment (jsumners) [reply package.json:33]: @Eomm why?

**Pull request closed without merging**